lib/requester/core.js

```diff
@@ -76,6 +76,26 @@ var dns = require('dns'),
         removeRefererHeader: 'removeRefererHeaderOnRedirect'
     },
 
+    /**
+     * Helper function to extract top level domain for the given hostname
+     *
+     * @private
+     *
+     * @param hostname
```

Comment:
```suggestion
     * @param {String} hostname
```
Also, add the description.